osu.Game/Overlays/Profile/ProfileHeader.cs

```diff
@@ -386,14 +387,17 @@ private void loadUser()
                 infoTextLeft.AddText(new DrawableJoinDate(user.JoinDate), boldItalic);
             }
 
+            infoTextLeft.NewLine();
+
             if (user.LastVisit.HasValue)
             {
-                infoTextLeft.NewLine();
                 infoTextLeft.AddText("Last seen ", lightText);
                 infoTextLeft.AddText(new DrawableDate(user.LastVisit.Value), boldItalic);
-                infoTextLeft.NewParagraph();
             }
 
+            infoTextLeft.NewParagraph();
+            infoTextLeft.NewLine();
```

Comment:
what's with this NewParagraphNewLine stuff?

Reply:
That code parses into an empty line. `NewParagraph` itself doesn't parse the empty line somehow.